docusaurus.config.js

```diff
@@ -171,11 +175,12 @@ const config = {
               {
                 label: 'GitHub',
                 href: 'https://github.com/fastify/website-next',
+                icon: '/img/icon/github.svg',
               },
             ],
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} My Project, Inc. Built with Docusaurus.`,
+        copyright: `Fastify, Copyright © 2016-${new Date().getFullYear()} <a href="https://openjsf.org">OpenJS Foundation<a> and The <a href="https://github.com/fastify/fastify#team">Fastify team</a>, Licensed under <a href="https://github.com/fastify/fastify/blob/main/LICENSE">MIT</a>`,
```

Comment:
Please, verify that the code shows what you would expect before pushing the code

I can't see the icons, and there is a typo that break the colors:

```diff
-ation<a>
+ation</a>
```

<img width="776" alt="image" src="https://user-images.githubusercontent.com/11404065/218265568-7a1bd286-a9be-4a39-ab9d-8ca13d0b6563.png">